tests/actions/test_core_actions.py

```diff
@@ -135,6 +137,45 @@ def test_PagerdutyAlertAction(
         data_asset=None,
     ) == {"pagerduty_alert_result": "none sent"}
 
+
+def test_MicrosoftTeamsNotificationAction(
+    data_context_parameterized_expectation_suite,
+    validation_result_suite,
+    validation_result_suite_extended_id,
+):
+    renderer = {
+        "module_name": "great_expectations.render.renderer.microsoft_teams_renderer",
+        "class_name": "MicrosoftTeamsRenderer",
+    }
+    teams_webhook = "https://webhook.site/700303fe-6398-4a22-96fb-837f90d019e9"
```

Comment:
Is this a special test token?

Reply:
It is an expiring url unfortunately:

- http://docs.webhook.site/premium.html#unlimited-requests
- http://docs.webhook.site/premium.html#non-expiring-urls

 We need to find a better way to deal wit this to run this test.

Reply:
@Antoninj one pattern you may be able to follow is the test for `PagerdutyAlertAction` (`test_PagerdutyAlertAction()` is in the same `test_core_actions.py` file)

For `PagerdutyAlertAction`, the `_run` function's response does not come from the actual the POST, but is `success` or `none sent` depending on whether the POST event sent as expected.   

Admittedly, this is not a 'full' test, but in combination with the `test_MicrosoftTeams_validation_results_with_datadocs()` test you wrote, I think we can be `reasonably` confident that notification will behave as expected.